docs/agents/tools.md

```diff
@@ -70,3 +70,23 @@ MCP can either be run locally ([MCPStdio][any_agent.config.MCPStdio]) or you can
         ]
     )
     ```
+
+## A2A tools
+
+`any-agent` provides a tool to wrap a connection to another another agent served over the A2A protocol, by invoking the `any_agent.tools.a2a_query` function, as shown in the following example:
+
+```python
+agent_cfg = AgentConfig(
+    instructions="Use the available tools to obtain additional information to answer the query.",
+    description="A sample agent.",
+    tools=[
+        await a2a_query(
```

Comment:
We should clarify with a note that this snippet requires to be run in async context and/or prioritize https://github.com/mozilla-ai/any-agent/issues/343